css-brochure-form/styles.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+h2 {
+  text-align: center;
+  font-family: 'Brandon Grotesque', sans-serif;  /* added fall back sans serif to all font-family properties */
+  font-style: normal;                            /* because git wasn't letting me commit */
+  font-weight: bolder;
+  font-size: 18px;
+  color: orangered;
+  margin-top: 20px;
+  margin-bottom: 8px;
+  letter-spacing: 1px;
+}
+
+form {
+  margin: auto;
+  width: 50%;
+}
+
+.box {
+  margin-bottom: 10px;
+  margin-top: 3px;
+  border-color: lightgray;
```

Comment:
You have no border on this element, so this is doing nothing. It can be removed.